editor.planx.uk/src/@planx/components/shared/Preview/Card.tsx

```diff
@@ -43,7 +43,12 @@ const Card: React.FC<Props> = ({
     path === ApplicationPath.SaveAndReturn && handleSubmit;
 
   return (
-    <Fade in={true} timeout={theme.transitions.duration.enteringScreen}>
+    <Fade
```

Comment:
my first instinct was to wrap this fade in `<Suspense fallback={<DelayedLoadingIndicator />}>`, but I couldn't actually ever get to the loading indicator to render between slow Card transitions - I think because Fade is not actually a React lazily-loaded component (even though the `mountOnEnter` prop is used to "lazy mount" the component), and therefore Suspense is never "activated" ? https://react.dev/reference/react/Suspense

very open to feedback if I'm missing something obvious here!